### ISSUE TITLE
causes server to get stuck when given stop command 

### ISSUE THREAD

**Issue by IchigoGames** [Szewek/Flux#22]
- **Minecraft:**1.15.2
- **Forge:**31.1.47 and 49
- **Flux Library:**fl-3.2.0

[latest.log](https://github.com/Szewek/Flux/files/4534015/latest.log)

**Steps to Reproduce:**
 1. ...use a server host that shows when server is stopped or stopping or type /stop on local server host 
 2. ...add mod fl-3.2.0

**Description of issue:**
to start out the issue i have been having with my modpack is something was keeping server from being able to save and then stop it would get stuck saving and then i had to kill it or wait for server to say it crashed when it simply did not respond for whatever the set time is so server killed task so no crash report as it was not a true crash after a lot of testing adding a mod a few mods at a time then starting and stooping test server i was able to find the mod that was doing it it fl-3.2.0
never expected it to be a core mod that was the issue but it is with this mod sever will not stop on its own without it it saves and stops fine. let me know if you need more info and logs.

**Comment by IchigoGames** [comment]
will start removing the other mods that are on server to make sure it does this by it self with no other mods.

**Comment by IchigoGames** [comment]
can confirm with this mod and no other mods on server it will not come to a stop 
![Capture](https://user-images.githubusercontent.com/11888162/80291382-a4119c00-8712-11ea-956c-2c35f7f50896.PNG)

**Comment by IchigoGames** [comment]
after waiting a few minutes server is forced to kill task.
![Captdddure](https://user-images.githubusercontent.com/11888162/80291419-079bc980-8713-11ea-9e56-08651219440a.PNG)

**Comment by IchigoGames** [comment]
[logs fl with no other mods.zip](https://github.com/Szewek/Flux/files/4534074/logs.fl.with.no.other.mods.zip)
here is logs with just this mod and no other mods.

**Comment by IchigoGames** [comment]
are you sure this is fixed im still haveing issues on my server not being able to stop and i am on fl-3.2.2

**Comment by Szewek** [comment]
The development server (vanilla with Forge) where I tested FL stopped almost immediately (before e9e05a3  it waited about 30 seconds before quitting. I tested it further with VisualVM and nothing else was keeping the server running.

I just tested production MC Forge server (with default settings). It also stops with a command without waiting. I even tested it in Multicraft server - same result. So it isn't a mod related issue anymore. I can't help with custom servers.
![Screenshot_2020-04-29 Multicraft - Console](https://user-images.githubusercontent.com/3748327/80642484-dac71980-8a66-11ea-9457-e06c29d85203.png)
